Rakefile

```diff
@@ -35,3 +36,19 @@ task :release_all do
     end
   end
 end
+
+desc "Generates a development app."
+task :development_app do
+  Dir.chdir("#{File.dirname(__FILE__)}") do
+    sh "rm -fR development_app"
+  end
+
+  Decidim::Generators::AppGenerator.start(
+    ["development_app", "--path", ".."]
```

Comment:
Not easy to parameterize this, as finding out the relative directory of the one you provided might be tricky.